Source/Core/Core/IOS/Network/SSL.cpp

```diff
@@ -35,6 +35,40 @@ static constexpr mbedtls_x509_crt_profile mbedtls_x509_crt_profile_wii = {
     0,         /* No RSA min key size */
 };
 
+namespace
+{
+// Dirty workaround to disable SNI.
```

Comment:
This comment should probably explain why SNI is being disabled.

Reply:
Done.